src/content/language/collections.md

```diff
@@ -242,78 +252,347 @@ final constantMap = const {2: 'helium', 10: 'neon', 18: 'argon'};
 For more information about maps, refer to the Maps section of the
 [`dart:core` documentation](/libraries/dart-core#maps).
 
-## Operators
+[Using constructors]: /language/classes#using-constructors
+[`Map`]: {{site.dart-api}}/dart-core/Map-class.html
+[type inference]: /language/type-system#type-inference
+
+## Collection elements {: #collection-elements }
+
+In a collection, an element is an individual item in that
+collection. These types of elements are supported in
+a collection:
```

Comment:
This is where the confusion of "element" that I mentioned particularly shows up.

The following h3 subsections are talking about elements in a collection literal, but this intro conflates items in actual collections and the elements in a literal. This mostly affects the usage of "individual item" as the collection elements can result in multiple items.

```suggestion
Collection elements are the syntactical constructs
used within collection literals to define the
resulting collection's initial contents.

These types of elements are supported in a collection:
```

Reply:
I have an alternative for us to take a look at tomorrow.

Reply:
Tentatively closing this because that section has changed so much. I'll have an addition revision today that we should look over when its complete. I'll let you know when it's ready.